src/client.js

```diff
@@ -67,13 +67,10 @@ export function getContainers(system, id) {
 }
 
 export function getContainerStats(system, callback) {
-    return new Promise((resolve, reject) => {
-        const options = {
-            stream: true,
-        };
-        podmanMonitor("libpod/containers/stats", "GET", options, callback, system)
-                .then(resolve, reject);
-    });
+    const options = {
+        stream: true,
+    };
+    return podmanMonitor("libpod/containers/stats", "GET", options, callback, system);
```

Comment:
I'd clean this up even further; declaring `options` here seems overkill. You can just make this all functional with something like
```js
export const getContainerStats = (system, callback) => podmanMonitor("libpod/containers/stats", "GET", { stream: true }, callback, system);
```

which is nice and concise IMHO.

Reply:
Fine by me, I wasn't sure if we wanted to go so far but happy to delete more lines :)

Reply:
Of course this is a good cleanup. This is just gonna be copy-pasta from when this whole file has been introduced. So please feel free to clean it up as well :)